dashboards-observability/common/constants/autocomplete.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const firstCommand = [{ label: 'source' }];
+
+export const pipeCommands = [
+{ label: 'dedup' },
```

Comment:
not issue. we might need something to format the files automatically, maybe a precommit hook, the indentations are inconsistent

Reply:
maybe like a linter??